### PR TITLE
Tech Debt: Cleanup MainActivty after merge with abstract CustomMainActivty

### PR DESCRIPTION
**Pull request by SchoolGuy**
After #228 I did go over the code of the newly merged class and found that some code can be removed because it is unused and thus not useful. I think it can safely go away as Android Studio pointed out to me.

### CODE REVIEW

**Comment by SchoolGuy** [comment]
Rebased onto master

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java [hunk 1]
```diff
@@ -443,36 +415,6 @@ protected boolean testFileSystem() {
         return true;
     }
 
-    protected boolean testFreeSpace() {
-        if (DIRS.length == 0)
-            return true;
```
Comment (bekuno) [inline src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java:448]: Why deleting this method?

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java [hunk 2]
```diff
@@ -385,33 +361,29 @@ public void showDialogFinish(final int typeOfFinish) {
...
-                        // Intent intent = new Intent();
-                        // intent.setAction(android.content.Intent.ACTION_VIEW);
-                        // intent.setDataAndType(Uri.fromFile(new File(FileSystem.ROOT + "smartmaps.apk")),
-                        // "application/vnd.android.package-archive");
-                        //
-                        // startActivity(intent);
-                        showDialogFinish(FINISH_EXIT_FORCE);
-                    }
+            b.setPositiveButton(R.string.ok, (dialog, which) -> {
+                if (finishType == FINISH_EXIT || finishType == FINISH_EXIT_FORCE) {
```
Comment (bekuno) [inline src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java:365]: No remove of obsolete code.
Please do the lambda migration it in another PR (or at least a separate commit and a suitable PR title).

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java (hunk 1 above)
Comment (SchoolGuy) [reply src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java:448]: It is nowhere used in our application

Comment (bekuno) [reply src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java:448]: It is used - in `onRequestPermissionsResult` and `onCreate`.

It should optimized instead.

Comment (SchoolGuy) [reply src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java:448]: You mean the method `testFileSystem` not this one.

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java (hunk 2 above)
Comment (SchoolGuy) [reply src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java:365]: I split the commit and addressed your desire of having another title.

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java (hunk 1 above)
Comment (bekuno) [reply src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java:448]: You are right. 
The `testFreeSpace` was deactivated in 70c1a930c1eb23c00b16f55fe346752f47b2ecc8.

Please check if now som strings are obsolete.

Comment (SchoolGuy) [reply src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java:448]: Found and removed

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java [hunk 3]
```diff
@@ -53,7 +52,6 @@
 import cz.matejcik.openwig.formats.CartridgeFile;
 import locus.api.objects.extra.Location;
 import locus.api.objects.extra.Waypoint;
-
```
Comment (bekuno) [inline src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java:56]: Please let the space before "import menion.*"

File: src/main/res/values/strings.xml [hunk 4]
```diff
@@ -66,7 +66,6 @@
     <string name="no_satellites">No satellites</string>
     <string name="no_target">No target!</string>
     <string name="no_wherigo_cartridge_available">No Wherigo cartridge (*.gwc) is available.\nDownload or copy them into directory \"%1$s\". You can also set a path to custom directory in \"Settings - Main - Wherigo folder\". Make sure you granted the application permission to write to external storage and you have write permission in selected directory. Some newer devices don\'t allow writing to SD card.If you\'re not sure, set \"Wherigo folder\" automatically.</string>
-    <string name="not_enough_disk_space_x">Not enough free disk space on SDCard!\n(%1$s, free:%2$s)</string>
```
Comment (bekuno) [inline src/main/res/values/strings.xml:69]: It would be enough to delete the string in the english source file.
The change for the other languages will be then done by the crowdin integration.
You can let it as is.

**Review by bekuno — COMMENTED** [review]
Beside the cosmetic change it looks fine.

File: src/main/res/values/strings.xml (hunk 4 above)
Comment (SchoolGuy) [reply src/main/res/values/strings.xml:69]: Thanks for the hint.

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java (hunk 3 above)
Comment (SchoolGuy) [reply src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java]: Fixed

**Comment by SchoolGuy** [comment]
@bekuno Should be fine now. Right?
